packages/cerebral/src/Controller.js

```diff
@@ -16,9 +17,11 @@ import {dependencyStore as computedDependencyStore} from './Computed'
   based on top level providers and providers defined in modules
 */
 class Controller extends EventEmitter {
-  constructor ({state = {}, signals = {}, providers = [], modules = {}, router, devtools = null, strictRender = false}) {
+  constructor ({state = {}, signals = {}, providers = [], modules = {}, router, devtools = null, options = {}}) {
     super()
-    this.strictRender = strictRender
+    this.computedDependencyStore = computedDependencyStore
```

Comment:
Let move instantiating of `computedDependencyStore` here, because now it seems not safe to have few controllers in one app since all of computeds will share one instance.
Is it possible to share computedDependencyStore instance from model and hoc?